spec/factories/vm_or_template.rb

```diff
@@ -14,10 +14,59 @@
     raw_power_state "never"
   end
 
-  factory(:vm,             :class => "Vm",            :parent => :vm_or_template)
-  factory(:vm_cloud,       :class => "VmCloud",       :parent => :vm)       { cloud true }
-  factory(:vm_infra,       :class => "VmInfra",       :parent => :vm)
+  factory(:vm, :class => "Vm", :parent => :vm_or_template)
+
+  factory(:vm_orphaned, :class => "Vm", :parent => :vm_or_template) do
+    ext_management_system nil
+    storage_id            123
```

Comment:
:-1:   Do not use hardcoded ids.  This in particular is wrong since the tests run on region 1, and this id can't possibly exists on that region.